### PR TITLE
feat: fetch user data at mount

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue
[[defect] 透過 menu 登入，存在會員資料不正常顯示](https://app.asana.com/0/1203699264568808/1206968504149584/f)

# Notice

# Dependency

### CODE REVIEW

File: src/containers/Member.js [hunk 1]
```diff
@@ -134,6 +136,7 @@ const MemberPage = ({
         window.location.href = getSignInHref(currentHref)
       }, 2000)
     }
+    getUserData(jwt, userID)
```
Comment (Aylie-Chou) [inline src/containers/Member.js:139]: could we add a blank before this line or add a comment to show its purpose ?
IMO, it looks like part of the check authorization section now 😂

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]
please have a look at comments ~~

File: src/containers/Member.js (hunk 1 above)
Comment (liruchen32) [reply src/containers/Member.js:140]: fixed

File: src/containers/Member.js [hunk 2]
```diff
@@ -298,14 +301,18 @@ MemberPage.propTypes = {
...
+  getUserData: PropTypes.func.isRequired,
 }
 
-const { reduxStateFields } = twreporterRedux
+const { actions, reduxStateFields } = twreporterRedux
+const { getUserData } = actions
 const mapStateToProps = state => {
   const jwt = _.get(state, [reduxStateFields.auth, 'accessToken'], '')
   const isAuthed = _.get(state, [reduxStateFields.auth, 'isAuthed'], false)
+  const userID = _.get(state, [reduxStateFields.auth, 'userInfo', 'user_id'])
```
[the remark this answers is not in the document]
Comment (liruchen32) [reply src/containers/Member.js:316]: 沒有呀

[the remark this answers is not in the document]
Comment (Aylie-Chou) [reply src/containers/Member.js:316]: 看太快了 😂
那是不是要給他一個 default value，因為他的 propsType 有 isRequired

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]
final check !

**Review by Aylie-Chou — APPROVED** [review]
LGTM

**Pull request merged**